src/utils/Theme/themes/dark.ts

```diff
@@ -110,6 +110,13 @@ export const darkColors: Colors = {
     700: 'rgba(0, 0, 0, 0.07)',
     800: 'rgba(0, 0, 0, 0.08)',
     900: 'rgba(0, 0, 0, 0.09)'
+  },
+  pink: {
+    500: '#281e21',
+    400: '#B4005E',
+    300: '#DB4274',
+    200: '#EB477C',
+    100: '#ff4d87'
```

Comment:
why are we defining new colors here? we should just use existing colors to match styling of the package